File: java/Human.java
public class Human {
    private int nik;
    private String nama, jenis_kelamin;

    public Human(){}

    //sett
    public void setNama(String nama){
        this.nama = nama;
    } 
    //get
    public String getNama(){
        return this.nama;
    }

    //sett
    public void setNik(int nik){
        this.nik = nik;
    } 
    //get
    public int getNik(){
        return this.nik;
    }

    //sett
    public void setJK(String jenis_kelamin){
        this.jenis_kelamin = jenis_kelamin;
    } 
    //get
    public String getJK(){
        return this.jenis_kelamin;
    }
}

File: c++/Human.cpp
#include<string>
#include<iostream>

using namespace std;

class Human{
    private:
        int nik;
        string nama, jenis_kelamin;
    
    public:
        Human(){

        }

        void setNik(int nik){
            this->nik = nik;
        }
        int getNik(){
            return this->nik;
        }

        void setNama(string nama){
            this->nama = nama;
        }
        string getNama(){
            return this->nama;
        }

        void setJenisKelamin(string jenis_kelamin){
            this->jenis_kelamin = jenis_kelamin;
        }
        string getJenisKelamin(){
            return this->jenis_kelamin;
        }

        ~Human(){
            
        }
};